src/rust-crypto/rust-crypto.ts

```diff
@@ -202,10 +202,23 @@ export class RustCrypto implements CryptoBackend {
     }
 
     public async exportRoomKeys(): Promise<IMegolmSessionData[]> {
-        // TODO
-        return [];
+        const raw = await this.olmMachine.exportRoomKeys(() => true);
+        return Promise.resolve(JSON.parse(raw));
+    }
+
+    public async importRoomKeys(keys: IMegolmSessionData[], opts?: IImportRoomKeysOpts | undefined): Promise<void> {
```

Comment:
The `undefined` is superfluous given the `?:`

Reply:
done